web/src/beta/features/Assets/hooks.ts

```diff
@@ -95,7 +96,7 @@ export default ({
   }, [endCursor, sort, fetchMore, hasMoreAssets, isGettingMore]);
 
   const handleSortChange = useCallback(
-    (type?: string, reverse?: boolean) => {
+    (type?: string | string[], reverse?: boolean) => {
```

Comment:
```suggestion
    (type?: string, reverse?: boolean) => {
```
I think for `type` it should not be an array. If this comes from Selector, checking for string or string[] should be handled before calling this.